### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [AsianEric/ML-exercise#1]
Good job on this assignment ! you mostly completed all the questions, that's a very good start. 
But here are something further:

** How many people have the job title of "Lawyer" ?**
Your logic is correct here, but a minor error:
- It is supposed to be `if 'lawyer' in job.lower():`, not `Lawyer`, since you turn the job title to be lowercase. 

Additionally:
- This question can be solved much more easily, by using simple filters. You can check it out in the solution. 

**How many people made the purchase during the AM and how many people made the purchase during PM ?**
- The `value_counts()` already gives you the correct answer. 

**How many people have American Express as their Credit Card Provider and made a purchase above $95 ?** You can solve this step by step:
- First let's filter the people with `CC Provider` to be ` American Express`
- Then select the people with `Purchase Price` to be over 95
- You can combine the above filters by using logical `&` operator in Python. 
- The full answer is given in the solution, you should review it later.

**Hard: How many people have a credit card that expires in 2025?**. This can be solved as:
- Create a function that is applied to the `CC Exp Date` column.
- This function check if the given year is 25 (`return True`) or not (`return False`).
- By returning `True` or `False`, we can calculate the number of `True` by using `sum()`.
- You can check the full answer in solution file.

**Hard: What are the top 5 most popular email providers/hosts (e.g. gmail.com, yahoo.com, etc...)**
- You create a function that is applied to `Email` column.
- This function just handles the task of extracting the email provider and return it.
- After applying this function to `Email` column, you can use `value_counts()` to return the number distinct email provider, and using `head()` to select the top 5.
- If you solve this, you can surely plot this one in the next question.

To sum up: This is a great start. Keep up the good work and have more practice!

**Comment by coderschoolreview** [comment]
Great work on everything ! You finish this assignment comprehensively.

Keep up the spirit!
